packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -72,52 +72,95 @@ contract FederatedAttestations is
   }
 
   function _isRevoked(address signer, uint256 time) internal view returns (bool) {
-    if (revokedSigners[signer] > 0 && revokedSigners[signer] >= time) {
-      return true;
-    }
-    return false;
+    require(time >= 0);
+    uint256 revokedOn = revokedSigners[signer];
+    return revokedOn > 0 && revokedOn <= time ? true : false;
```

Comment:
nit
```suggestion
    return revokedOn > 0 && revokedOn <= time;
```

Reply:
👍  good catch! One note here is all the time logic has been removed in Izzy's work on this (since corrupt signers can issue revocations in the past)